aws/resource_aws_waf_web_acl_test.go

```diff
@@ -275,6 +275,80 @@ resource "aws_waf_web_acl" "test" {
 `, rName, rName)
 }
 
+func testAccAWSWafWebAclConfig_Logging(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_waf_web_acl" "test" {
+  metric_name = %q
+  name        = %q
+
+  default_action {
+    type = "ALLOW"
+  }
+
+  logging_configuration {
+
+    log_destination = "${aws_kinesis_firehose_delivery_stream.test_stream.arn}"
+
+    redacted_fields {
+
+      field_to_match {
+        type = "QUERY_STRING"
+      }
+
+      field_to_match {
+        type = "METHOD"
+      }
+
+      field_to_match {
+        type = "URI"
+      }
+
+      field_to_match {
+        type = "HEADER"
+        data = "referer"
+      }
+
+    }
+  }
+}
+resource "aws_s3_bucket" "bucket" {
+  bucket = "tf-test-bucket-%d"
+  acl    = "private"
+}
+
+resource "aws_iam_role" "firehose_role" {
+  name = "tf_acctest_firehose_delivery_role_%d"
+
+  assume_role_policy = <<EOF
+{
+  "Version": "2012-10-17",
+  "Statement": [
+    {
+      "Action": "sts:AssumeRole",
+      "Principal": {
+        "Service": "firehose.amazonaws.com"
+      },
+      "Effect": "Allow",
+      "Sid": ""
+    }
+  ]
+}
+EOF
+}
+
+resource "aws_kinesis_firehose_delivery_stream" "test_stream" {
+  name        = "terraform-kinesis-firehose-waftest-%d"
```

Comment:
The name for WAF logging Kinesis Firehose Delivery Streams must begin with `aws-waf-logs-`

```suggestion
  name        = "aws-waf-logs-%s"
```